src/main/scala/controllers/OpportunityController.scala

```diff
@@ -49,6 +49,53 @@ class OpportunityController @Inject()(opportunities: OpportunityOps, application
     }.getOrElse(Redirect(controllers.routes.OpportunityController.showNewOpportunityForm()))
   }
 
+  def viewTitle(id:OpportunityId) = Action.async { request =>
```

Comment:
There are a lot of new actions going into `OpportunityController`. I think we should go with the idea of having a new `controllers.manage` package and move all of these to an `OpportunityController` under that package to keep the actions separate from the applicant journey and parallel what is happening in the `views`.